addons/custom-fonts/addon.json

```diff
@@ -0,0 +1,145 @@
+{
+  "name": "Customizable fonts",
+  "description": "Customize the font family for different parts of Scratch.",
+  "tags": ["theme", "beta", "editor"],
+  "credits": [
+    {
+      "name": "ErrorGamer2000",
+      "link": "https://scratch.mit.edu/users/ErrorGamer2000/"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "id": "bunchwarn",
+      "text": "Letters in wider fonts will be bunched together. Try not to use excessively large fonts."
+    }
+  ],
+  "enabledByDefault": false,
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "versionAdded": "1.24.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/*"],
+      "runAtComplete": true
+    }
+  ],
+  "settings": [
+    {
+      "name": "Navigation bar font",
+      "id": "navfnt",
+      "type": "datalist",
+      "default": "Helvetica",
+      "items": [
+        "Helvetica",
+        "Arial",
+        "Roboto",
+        "Lucida Grande",
+        "Verdana",
+        "Times New Roman",
+        "Sora",
+        "DejaVu Sans",
+        "Handlee",
+        "Antonio",
+        "Lato"
+      ]
+    },
+    {
+      "name": "Main font",
+      "id": "mainfnt",
+      "type": "datalist",
+      "default": "Helvetica",
+      "items": [
+        "Helvetica",
+        "Arial",
+        "Roboto",
+        "Lucida Grande",
+        "Verdana",
+        "Times New Roman",
+        "Sora",
+        "DejaVu Sans",
+        "Handlee",
+        "Antonio",
+        "Lato"
+      ]
+    },
+    {
+      "name": "Editor font",
+      "id": "editfnt",
+      "type": "datalist",
+      "default": "Helvetica",
+      "items": [
+        "Helvetica",
+        "Arial",
+        "Roboto",
+        "Lucida Grande",
+        "Verdana",
+        "Times New Roman",
+        "Sora",
+        "DejaVu Sans",
+        "Handlee",
+        "Antonio",
+        "Lato"
+      ]
+    },
+    {
+      "name": "Block Font",
+      "id": "blockfnt",
+      "type": "datalist",
+      "default": "Helvetica",
+      "items": [
+        "Helvetica",
+        "Arial",
+        "Roboto",
+        "Lucida Grande",
+        "Verdana",
+        "Times New Roman",
+        "Sora",
+        "DejaVu Sans",
+        "Handlee",
+        "Antonio",
+        "Lato"
+      ]
+    },
+    {
+      "name": "Header font",
```

Comment:
This name is confusing: especially because another setting is called "Footer font", I thought this would apply to the editor menu bar or something else at the top of a page. Box headers are an outdated design element that is only found on old pages (2015 and earlier), so I don't think they need a separate setting. A "heading font" that would apply to all h# elements would be much more useful.

Reply:
> Box headers are an outdated design element that is only found on old pages (2015 and earlier)

Actually, no. They are on the front page, and they got a new class name.

Reply:
The current front page was designed in 2015.

Reply:
Yeah, they are on the front page still, but I set it to include all `h#` elements too.